pages/contribute/how_to_contribute.md

```diff
@@ -1,39 +1,30 @@
 ---
 title: How to contribute to RDMkit
+toc: false
 ---
 
-## Ways of contributing
+This project is only possible with the many [community contributors](https://rdmkit.elixir-europe.org/contributors). RDMkit is an open project where everyone can contribute to the site.
 
-Everyone is welcome to contribute to this site, and we've tried to make it as easy as possible. Choose a way that suits you. The website is hosted on GitHub, so:
+If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency: 
```

Comment:
> If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency:

If you wish to contribute, please:

Maybe we can keep this shorter, and not repeat that it is a community project.